### PR TITLE
PAS-11 armor and M10 helmets are now modular.

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
PAS-11 armor and M10 helmets have been replaced by the PAS-11X and M10X variants, which have been fit with attach points allowing them to accept modules, however, the PAS-11X doenst take different armor plates, its just the default. 

## Why It's Good For The Game
Removes the gap and difference between PAS-11/M10 with Jaeger by making them able to take modules, now they are the same thing and have pretty much the same balance.

## Changelog
:cl:
expansion: PAS-11 armor and M10 helmet have been fitted with attach points to be able to take modules like jaeger, a X has been added on end of their name to imply that.
balance: PAS-11X and M10X can take modules.
/:cl:

### CODE REVIEW

**Comment by token-map** [comment]
dangerously based

**Comment by ilikesaying** [comment]
based

File: code/__DEFINES/loadout.dm [hunk 1]
```diff
@@ -446,7 +446,7 @@ GLOBAL_LIST_INIT(leader_clothes_listed_products, list(
 		/obj/item/helmet_module/welding = list(CAT_HEL, "Jaeger welding module", 0, "orange"),
 		/obj/item/helmet_module/binoculars =  list(CAT_HEL, "Jaeger binoculars module", 0, "orange"),
 		/obj/item/helmet_module/antenna = list(CAT_HEL, "Jaeger Antenna module", 0, "orange"),
-		/obj/item/clothing/head/helmet/marine/leader  = list(CAT_HEL, "Regular leader helmet", 0, "black"),
+		/obj/item/clothing/head/modular/marine/m10x/leader  = list(CAT_HEL, "Regular leader helmet", 0, "black"),
```
Comment (GradusNL) [inline code/__DEFINES/loadout.dm:449]: Shouldn't this be M10X leader helmet to bring it in line with the others?

**Comment by neohenry** [comment]
When greyscale for PAS?

**Comment by SplinterGP** [comment]
> 
> 
> When greyscale for PAS?

suffer

**Comment by Kuro020** [comment]
> When greyscale for PAS?

after this is merged

File: code/__DEFINES/loadout.dm (hunk 1 above)
Comment (SplinterGP) [inline code/__DEFINES/loadout.dm:449]:
```suggestion
		/obj/item/clothing/head/modular/marine/m10x/leader  = list(CAT_HEL, "M10X leader helmet", 0, "black"),
```

**Comment by Tristan63** [comment]
attachable armor plate for m10x?

**Comment by Ieteihowmee** [comment]
PAS seperated the men from the boys. I am not sure how I feel about this. Though I am curious to see how it all works. Can I get Mimir on my PAS? I certainly hope so :)

**Comment by SplinterGP** [comment]
> 
> 
> PAS seperated the men from the boys. I am not sure how I feel about this. Though I am curious to see how it all works. Can I get Mimir on my PAS? I certainly hope so :)

ye

**Review by David-Stormwell — APPROVED** [review]
Approved from a balance standpoint, also required later so that MJP or anyone else can replace the sprites with better sprites while having some modularity.

File: code/modules/modular_armor/modular.dm [hunk 2]
```diff
@@ -338,15 +355,9 @@
...
-	item_state = list(
-		slot_head_str = ""
-	)
+
 	greyscale_config = /datum/greyscale_config/modularhelmet_infantry
 	greyscale_colors = "#5B6036#f7fb58"
-	//head icon is generated so null this
-	item_state_slots = list(
-		slot_head_str = null,
-	)
```
Comment (TiviPlus) [inline code/modules/modular_armor/modular.dm:349]: why are you deleting head icons

File: code/modules/modular_armor/modular.dm [hunk 3]
```diff
@@ -636,3 +651,47 @@
...
+	icon = 'icons/mob/modular/m10.dmi'
+	icon_state = "helmet_icon"
+	icon_override = null
+	item_state = "helmet"
+	item_state_worn = TRUE
+	item_state_slots = null
+	item_icons = list(
+		slot_head_str = 'icons/mob/modular/m10.dmi',
+		slot_l_hand_str = 'icons/mob/items_lefthand_1.dmi',
+		slot_r_hand_str = 'icons/mob/items_righthand_1.dmi',)
```
Comment (TiviPlus) [inline code/modules/modular_armor/modular.dm:667]: formatting

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Show pictures

File: code/modules/modular_armor/modular.dm (hunk 3 above)
Comment (SplinterGP) [inline code/modules/modular_armor/modular.dm:667]:
```suggestion
	item_icons = list(
		slot_head_str = 'icons/mob/modular/m10.dmi',
		slot_l_hand_str = 'icons/mob/items_lefthand_1.dmi',
		slot_r_hand_str = 'icons/mob/items_righthand_1.dmi',
	)
```

File: code/modules/modular_armor/modular.dm (hunk 2 above)
Comment (SplinterGP) [reply code/modules/modular_armor/modular.dm:349]: it being null doenst do anything.

**Comment by mathis650** [comment]
based

**Comment by rusher210** [comment]
totally based

**Review by BraveMole — APPROVED** [review]
No bug reports

**Pull request merged**